assets/js/modules/sign-in-with-google/datastore/anyoneCanRegister.js

```diff
@@ -0,0 +1,127 @@
+/**
+ * `modules/sign-in-with-google` data store: get "Anyone can register" value from WordPress.
```

Comment:
As mentioned before, all of the infra around this value can likely be simplified into an extension of base data exposed via `core/site`.

Reply:
Removed all of this; it simplifies things a lot 👍🏻